<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatList</class>
 <widget class="QMainWindow" name="ChatList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0" colspan="2">
     <widget class="QPushButton" name="personal_chat">
      <property name="text">
       <string>Personal Chat</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QPushButton" name="create_group">
      <property name="text">
       <string>Create Group</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QPushButton" name="join_group">
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QListWidget" name="chat_list"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="UserID">
      <property name="text">
       <string>User ID:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="Value">
      <property name="text">
       <string>Value</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
